bigbluebutton-html5/imports/ui/components/chat/service.js

```diff
@@ -45,36 +42,49 @@ const mapUser = user => ({
   isLocked: user.locked,
 });
 
+const getUser = (userID) => {
```

Comment:
Please use `userId` instead of `userID`. Then you can simplify the selector as `{ userId }`